internal/sbi/producer/nf_management.go

```diff
@@ -230,10 +230,12 @@ func RemoveSubscriptionProcedure(subscriptionID string) {
 }
 
 func GetNFInstancesProcedure(nfType string, limit int) (*nrf_context.UriList, *models.ProblemDetails) {
-	// nfType := c.Query("nf-type")
-	// limit, err := strconv.Atoi(c.Query("limit"))
 	collName := "urilist"
 	filter := bson.M{"nfType": nfType}
+	if nfType == "" {
+		// if query parameter is not append, the nftype defalue doesn't use.
```

Comment:
Nitpick: I don't understand this comment. Do you mean something like "if the query parameter is not present, do not filter by nfType"?

Reply:
@GKnirps 
Yes, MongoAPI will return all of the documents if the filter is empty.

Reply:
@BENHSU0723 
Please fix this comment.